scripts/ci/run-ci-tests.sh

```diff
@@ -326,3 +326,8 @@ make -C test/others/action-script run
 
 # compel testing
 make -C compel/test
+
+# amdgpu_plugin testing
+make amdgpu_plugin
+make -C plugins/amdgpu/ test_topology_remap
+./plugins/amdgpu/test_topology_remap
```

Comment:
What does this test do? How does it work w/o amd devices?

Reply:
This is a unit test introduced in commit https://github.com/checkpoint-restore/criu/commit/ba9c62df24160743d8082cfc5f6af4e92a4e629a. There is a comment describing what the test does and how it works in [plugins/amdgpu/tests/test_topology_remap.c](https://github.com/checkpoint-restore/criu/blob/5de9040ee758f1fd1a2599b6f800013544c966b6/plugins/amdgpu/tests/test_topology_remap.c#L2).